assets/js/admin/form-builder.js

```diff
@@ -222,6 +223,46 @@
 			}
 		},
 
+		/**
+		 * Initialize date pickers like min/max date, disable dates etc.
+		 *
+		 * @since 1.7.0
+		 */
+		init_datepickers: function() {
+			var date_format = $( '.everest-forms-disable-dates' ).data( 'date-format' );
+			var selection_mode = 'multiple';
+
+			// Initialize `Disable dates` option's date pickers that hasn't been initialized.
+			$( '.everest-forms-disable-dates' ).each( function() {
+				if ( ! $( this ).get(0)._flatpickr ) {
+					$( this ).flatpickr({
+						dateFormat: date_format,
+						mode: selection_mode,
+					});
+				}
+			})
+
+			// Reformat the selected dates input value for `Disable dates` option when the date format changes.
+			$( document.body ).on( 'change', '.evf-date-format', function( e ) {
+				var $disable_dates = $( '.everest-forms-field-option:visible .everest-forms-disable-dates' );
```

Comment:
Please initialize all the vars in a single statement. they don't all need to be separate statements, anyways, the single statement one gets better optimized by the JIT Browser Engines, I'm told.

Reply:
Resolved with e3051942f7b54bebc0ba16ae0399987ede193104